Framework/Algorithms/test/PolarizationEfficiencyCorTest.h

```diff
@@ -486,10 +427,35 @@ class PolarizationEfficiencyCorTest : public CxxTest::TestSuite {
     return names;
   }
 
+  std::shared_ptr<PolarizationEfficiencyCor> createAlgorithm() {
+    auto alg = std::make_shared<PolarizationEfficiencyCor>();
+    alg->setRethrows(true);
+    alg->initialize();
+    return alg;
+  }
+
+  void setAlgorithmProperties(const std::shared_ptr<PolarizationEfficiencyCor> &alg,
+                              const std::string &efficiencyMethod, const std::string &method = "",
+                              const std::string &analysisMethod = "") {
+    alg->setProperty("OutputWorkspace", "out");
+    if (!method.empty()) {
+      alg->setProperty("CorrectionMethod", method);
+    }
+    alg->setProperty("Efficiencies", createEfficiencies(efficiencyMethod));
+    if (!analysisMethod.empty()) {
+      alg->setProperty("PolarizationAnalysis", analysisMethod);
+    }
+  }
+
+  void checkWorkspaceGroupSize(size_t expectedSize) {
+    WorkspaceGroup_sptr out = AnalysisDataService::Instance().retrieveWS<WorkspaceGroup>("out");
```

Comment:
Could we define "out" as a constant i.e. `const std::string OUTPUT_GRP_NAME{"out"};`. We use the value in a couple of places, so it would be nice to have it defined in just one place if we can.